packages/web/src/pages/feed-page/components/mobile/FeedPageContent.tsx

```diff
@@ -54,6 +57,7 @@ const FeedPageMobileContent = ({
         />
       </Header>
     )
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
just wanted to make sure it makes sense to not use all deps

Reply:
I was able to add the dep, there was an issue with `useModalState`